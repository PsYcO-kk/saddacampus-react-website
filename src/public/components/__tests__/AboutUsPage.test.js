import {shallow} from 'enzyme';
import React from 'react';
import AboutUsPage from '../AboutUsPage';

test("should render AboutUsPage correctly" , ()=>{
    const wrapper = shallow(<AboutUsPage />)
    expect(wrapper).toMatchSnapshot();
})
