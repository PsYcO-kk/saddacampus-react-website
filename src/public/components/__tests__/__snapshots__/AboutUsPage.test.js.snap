// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AboutUsPage correctly 1`] = `
<div>
  <h1>
    About Us
  </h1>
</div>
`;
